blockchain/storage/leveldb/leveldb.go

```diff
@@ -20,12 +61,105 @@ func Factory(config map[string]interface{}, logger hclog.Logger) (storage.Storag
 		return nil, fmt.Errorf("path is not a string")
 	}
 
-	return NewLevelDBStorage(pathStr, logger)
+	// get the options
+	opt := NewDefaultOptons()
+
+	cache, ok := config["cache"]
```

Comment:
Can we use constants here?

Every little steps you take leads to a better future. 🙏

Reply:
This method was never used and I removed it